modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/macros/PRMHomePage.macro

```diff
@@ -1,5 +1,13 @@
 definition {
 
+	@summary = "Verify if the Announcement is visible on listing page"
+	macro gotoAnnouncementList {
+		AssertElementPresent(
+			key_announcementContent = "Announcement Body",
+			key_announcementTitle = "Announcement Title",
+			locator1 = "PRMAnnouncements#ANNOUNCEMENT_TITLE_AND_CONTENT");
+	}
```

Comment:
Hi @marinaasant0s, I think that this macro name don't match with the macro really do. Usually we use 'goto' when we are navigating to somewhere. The macro name could be 'viewAnnouncementOnList', since you are doing an assert, and not navigating. What do you think?